odxtools/nameditemlist.py

```diff
@@ -177,20 +176,11 @@ def __repr__(self) -> str:
         return f"{type(self).__name__}([{', '.join([repr(x) for x in self])}])"
 
     def __copy__(self) -> Any:
-        return self.__class__(list(self))
-
-    def __deepcopy__(self, memo: dict[int, Any]) -> Any:
-        cls = self.__class__
-        result = cls.__new__(cls)
-        memo[id(self)] = result
-        result._item_dict = {}
-        for x in self:
-            result.append(deepcopy(x, memo))
-
-        return result
+        return self.__class__(self)
```

Comment:
Is there a test case covering this?

Are you sure about changing `self.__class__(list(self))` to `self.__class__(self)` ?

Reply:
a simple test would be to mutate the original and see if the copy gets mutated